atd-vzd/migrations/default/1717076272414_fix_data_issues/up.sql

```diff
@@ -0,0 +1,36 @@
+update public.atd_txdot_crashes set surf_type_id = 99
+where surf_type_id = 'null';
+
+alter table lookups.unit_desc_lkp
+drop constraint unit_desc_owner_check,
+add constraint unit_desc_owner_check check (
+    (
+        (id < 177 and source = 'cris')
+        or (id >= 177 and source = 'vz')
+        or (id = 94 and source = 'vz')
+    )
+);
+
+
+alter table lookups.injry_sev_lkp
+drop constraint injry_sev_owner_check,
+add constraint injry_sev_owner_check check (
+    (
+        (id < 99 and source = 'cris')
+        or (id >= 99 and source = 'vz')
+        or (id = 94 and source = 'vz')
+    )
+);
```

Comment:
We can rework these constraints to not have this third, "poke a hole through the rule" id that we allow for VZ, but I sort of am thinking that this is the most minimally changing way to do this, and I like not claiming ids less than 100 for VZ any more than I have to, since CRIS has a history of using the "high hundreds" id values. I'm open to suggestion on this, big time.